mscs

```diff
@@ -22,18 +22,44 @@ if [ -r $MSCS_DEFAULTS ]; then
   MSCS_ARGS="-c $MSCS_DEFAULTS "
 fi
 
+# gosu for easy step-down from root
+# see: https://github.com/tianon/gosu
+GOSU=${GOSU:-"/usr/local/bin/gosu"}
+
 # Setup the default user name.
 USER_NAME=${USER_NAME:-minecraft}
 
 # Setup the default installation location.
 LOCATION=${LOCATION:-"/opt/mscs"}
 
+#Check permissions LOCATION and some top directories
+if [ -n "$(find $(find $LOCATION -maxdepth 2) \! -user "$USER_NAME" -print -prune -o -prune)" ]; then
+  echo "Note: Wrong owner of the directory, fixing"
+  if [ "$(whoami)" = "root" ]; then
+    chmod -R u+w $LOCATION && chown -R minecraft:minecraft $LOCATION
+  else
+    sudo chmod -R u+w $LOCATION && sudo chown -R minecraft:minecraft $LOCATION
+  fi
+fi
+
```

Comment:
This was added to the troubleshooting section of the readme, and the Makefile was updated to fix this problem on install.  I don't think this needs to be here.